src/plots/cartesian/layout_attributes.js

```diff
@@ -633,6 +633,17 @@ module.exports = {
         editType: 'ticks',
         description: 'Determines whether or not the tick labels are drawn.'
     },
+    labelalias: {
+        valType: 'any',
+        dflt: false,
+        editType: 'ticks',
+        description: [
+            'Determines the alias for certain labels for ticks and hover.',
+            'For example using {US: \'USA\', CA: \'Canada\'} maps US to USA',
+            'and CA to Canada if tick or hover labels match US or Canada i.e.',
+            'after adding prefix and suffix to labels.'
```

Comment:
```suggestion
            'Replacement text for specific tick or hover labels.',
            'For example using {US: \'USA\', CA: \'Canada\'} changes US to USA',
            'and CA to Canada. The labels we would have shown must match',
            'the keys exactly, after adding any tickprefix or ticksuffix.',
            'labelalias can be used with any axis type, and both keys (if needed)',
            'and values (if desired) can include html-like tags or MathJax.'
```

And to that last point, can we include some tags (`ZERO<sup>2</sup>` or something) and math in a couple of the updated mocks?

Reply:
Addressed in [2b75c3c](https://github.com/plotly/plotly.js/pull/6481/commits/2b75c3cf1ad4ca5eb8ac35951971f62b47b2980f) and [c83ac53](https://github.com/plotly/plotly.js/pull/6481/commits/c83ac53801e14db81ef1e0fef0d9269da116015c).